samples/routing/custom/Core_6/Billing/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.CustomRouting.Billing";
```

Comment:
Nitpicking - that could be replaced with constant (doesn't need another review if you'll do it).